demo/addons/godotopenxr/.export/godot_openxr_editor_plugin.gd

```diff
@@ -15,7 +15,7 @@ func _enter_tree():
 	meta_export_plugin = preload("meta/godot_openxr_meta_editor_export_plugin.gd").new("meta", plugin_version)
 	pico_export_plugin = preload("pico/godot_openxr_pico_editor_export_plugin.gd").new("pico", plugin_version)
 	lynx_export_plugin = preload("lynx/godot_openxr_lynx_editor_export_plugin.gd").new("lynx", plugin_version)
-	khr_export_plugin = preload("khr/godot_openxr_khr_editor_export_plugin.gd").new("khr", plugin_version)
+	khr_export_plugin = preload("khr/godot_openxr_khr_editor_export_plugin.gd").new("khronos", plugin_version)
```

Comment:
You would also need to update the folder names and the maven central dependency from 'khr' to 'khronos' to match, as the `vendor` name argument is used to determine the location of the aar binaries and of the maven central dependencies.

Reply:
I think I got it all now.